test_cases/reverse_address.json

```diff
@@ -0,0 +1,166 @@
+{
+  "name": "reverse non-coarse",
+  "priorityThresh": 1,
+  "endpoint": "reverse",
+  "tests": [
+    {
+      "id": 1,
+      "status": "pass",
+      "user": "trescube",
+      "description": "there should be no non-address results, lat/lon is for Lancaster, PA",
+      "type": "dev",
+      "in": {
+        "point.lat": 40.038123,
+        "point.lon": -76.304096
+      },
+      "expected": {
+        "properties": [
+          {
+            "layer": "address"
+          }
+        ]
+      },
+      "unexpected": {
+        "properties": [
+          {
+            "layer": "locality"
+          },
+          {
+            "layer": "localadmin"
+          }
+        ]
+      }
+    },
+    {
+      "id": 2,
+      "status": "pass",
+      "user": "trescube",
+      "description": "there should be no non-address results, lat/lon is for Ventura County, CA",
+      "type": "dev",
+      "in": {
+        "point.lat": 34.35755,
+        "point.lon": -119.126008
+      },
+      "expected": {
+        "properties": [
+          {
+            "layer": "street"
+          }
+        ]
+      },
+      "unexpected": {
+        "properties": [
+          {
+            "layer": "county"
+          }
+        ]
+      }
+    },
+    {
+      "id": 3,
+      "status": "pass",
+      "user": "trescube",
+      "description": "addresses should be returned before coarse results, this lat/lon is for Red Lion, PA",
+      "type": "dev",
+      "in": {
+        "point.lat": 39.898111,
+        "point.lon": -76.607958
+      },
+      "expected": {
+        "properties": [
+          {
+            "layer": "address"
+          }
+        ]
+      },
+      "unexpected": {
+        "properties": [
+          {
+            "gid": "whosonfirst:locality:101717221"
```

Comment:
There are some non-coarse reverse tests in https://github.com/pelias/acceptance-tests/blob/master/test_cases/reverse_coordinate_wrapping.json but they're for coordinate wrapping and don't actually verify any non-coarse fields.  

I've been putting gids in [some](https://github.com/pelias/acceptance-tests/blob/master/test_cases/placeholder_general.json) [places](https://github.com/pelias/acceptance-tests/blob/master/test_cases/placeholder_sorting.json).  The nice thing about gids is that they lessen the effect of language changes and we [filter out superseded records](https://github.com/pelias/whosonfirst/blob/master/src/components/isActiveRecord.js#L8-L12) during the import process so if a record was superseded then the test would fail.  But I will switch up the `unexpected` to just check for `locality` layer.